client/src/components/sendRequsetModel/Form.tsx

```diff
@@ -0,0 +1,261 @@
+import {
+  Button,
+  FormControlLabel,
+  MenuItem,
+  Radio, RadioGroup, Select, TextField, Typography,
+} from '@mui/material';
+import { useFormik } from 'formik';
+import * as yup from 'yup';
+import brands from '../../brand.json';
+
+const validationSchema = yup.object({
+  brand: yup
+    .string()
+    .required('brand is required'),
+  model: yup
+    .string()
+    .required('model is required'),
+  year: yup
+    .number()
+    .required('year is required'),
+  Mileage: yup
+    .number()
+    .required('mileage is required'),
```

Comment:
```suggestion
  mileage: yup
    .number()
```
milage is not required at `db, also it's milage not Milage so rename it